.azure/azure-release-pipeline.yml

```diff
@@ -11,46 +11,80 @@ stages:
               python.version: '3.8'
           maxParallel: 1
       steps:
+      - script: |
```

Comment:
can we not name these scripts to see a summary of what each does?

Reply:
The name goes below the script (see line 48) by convention with Azure